### ISSUE TITLE
OpenDNS response IPs

### ISSUE THREAD

**Issue by geek-at** [dnlongen/Snort-DNS#1]
Hello! I tried to query some domains with the openDNS servers but none of them returned the IPs you have mentioned.

These are the official OpenDNS block page IPs:
[![OpenDNS block page IPs](https://www.pictshare.net/238cd3ae70.jpg)](https://support.opendns.com/entries/69715954-What-are-the-OpenDNS-Block-Page-IP-Addresses-)

The ones in this project don't seem to work (any more)

**Comment by dnlongen** [comment]
Hey - thanks for the note. When I wrote the demo, OpenDNS did not publish
official block page IPs so I had to do a little investigating; it seems
they have changed some things since then, I'll add an update to the project.

Is that table published somewhere on the OpenDNS site? I'd like to add a
link to the official reference.

Regards,
David Longenecker

Connect: Blog http://securityforrealpeople.com | @dnlongen
https://www.twitter.com/dnlongen | LinkedIn
https://www.linkedin.com/in/dnlongen/
PGP key: https://keybase.io/dnlongen
